src/Blazor.Extensions.Canvas/Canvas2dContext.cs

```diff
@@ -216,8 +221,8 @@ internal Canvas2dContext(BECanvasComponent canvasReference)
         public void StrokeRect(double x, double y, double width, double height) => this.CallMethod<object>(STROKE_RECT_METHOD, new object[] { x, y, width, height });
         public void FillText(string text, double x, double y, double? maxWidth = null) => this.CallMethod<object>(FILL_TEXT_METHOD, maxWidth.HasValue ? new object[] { text, x, y, maxWidth.Value } : new object[] { text, x, y });
         public void StrokeText(string text, double x, double y, double? maxWidth = null) => this.CallMethod<object>(STROKE_TEXT_METHOD, maxWidth.HasValue ? new object[] { text, x, y, maxWidth.Value } : new object[] { text, x, y });
-        public TextMetrics MeasureText(string text) => this.CallMethod<TextMetrics>(MEASURE_TEXT_METHOD, new object[] { text });
-        public float[] GetLineDash() => this.CallMethod<float[]>(GET_LINE_DASH_METHOD);
+        public ValueTask<TextMetrics> MeasureText(string text) => this.CallMethod<TextMetrics>(MEASURE_TEXT_METHOD, new object[] { text });
```

Comment:
good call! should we rename to MeasureTextAsync?

Reply:
You are using here methods versions that are already deprecated on the Blazor.Extensions.Canvas. Almost the call are with the Async suffix. But there is something that is causing the server side been very slow with a hang starting. In my opinion we shouldn't use it, because is not bring any advantages related with the original.